jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -36,7 +36,8 @@ class OutlierDetectionStep(Step):
         good_bits = integer(default=4)
     """
     reference_file_types = ['gain', 'readnoise']
-
+    prefetch_references = False
```

Comment:
I think we want this to set `prefetch_references = True` for the general case of outlier detection, yes?